march_hardware/include/march_hardware/PDOmap.h

```diff
@@ -6,36 +6,35 @@
 #include <utility>
 #include <vector>
 #include <map>
+#include <unordered_map>
 
 #include <ros/ros.h>
 
 #include <march_hardware/EtherCAT/EthercatSDO.h>
 
 namespace march4cpp
 {
+/** Store IMC data as a struct to prevent data overlap.*/
 struct IMCObject
 {
   int address;  // in IMC memory (see IMC manual)
   int length;   // bits (see IMC manual)
```

Comment:
These should be `uint16_t` and `const` to prevent any bugs by editing these values in the algorithm

Reply:
indeed!

Reply:
Ah yes `const` is not possible, since the other constructor still exists. But if you initialize the variables in the initializer list then can be const since you never want to edit it.

When I tried removing the constructor it errors on https://github.com/project-march/hardware-interface/blob/b0373bd305d7371fdb2d4e53e9291d47ca8e73a0/march_hardware/src/PDOmap.cpp#L40

For some reason this line uses the default constructor, so let's leave the `const` for now, but still change the type

Reply:
I've encountered the same problem. weird though. I'll leave it like it is.